lua/fugit2/view/git_status.lua

```diff
@@ -1687,8 +1687,11 @@ function GitStatus:_init_branch_menu()
       if vim.fn.exists ":Telescope" then
         self:unmount()
         vim.cmd { cmd = "Telescope", args = { "git_branches" } }
+      elseif vim.fn.exists ":FzfLua" then
```

Comment:
I'm still getting `Command not found: Telescope` with these changes.

Looks like `vim.fn.exists` returns `0` if the command doesn't exist, which lua considers truthy so we're still trying the `Telescope` cmd.

I think we have to check `vim.fn.exists ... > 0` on this line and above

Reply:
@ryancobb, you need configure your Telescope like this

```lua
  {
    'nvim-telescope/telescope.nvim',
    branch = '0.1.x',
    dependencies = {
      {
        'nvim-lua/plenary.nvim'
      },
      {
        'nvim-telescope/telescope-fzf-native.nvim',
        build = 'make'
      },
      {
        'nvim-telescope/telescope-file-browser.nvim'
      },
      {
        'nvim-telescope/telescope-frecency.nvim'
      },
    },
    cmd = { 'Telescope' },
    config = function() require 'config.telescope' end,
  }
```

The important part is `cmd = { 'Telescope' }`. So the `vim.fn.exists ":Telescope"` can return True. The same also apply for FzfLua

Reply:
Sorry @SuperBo, I should have provided more info!

This is occurring when I have have telescope disabled and am trying to use FzfLua for the branch menu.

For troubleshooting:
1. `:Telescope` returns `Telescope E492: Not an editor command: Telescope`
2. `:FzfLua` opens FzfLua
3. Trying to open fugit2 branch menu:
```
E5108: Error executing lua: ...are/nvim/lazy/fugit2.nvim/lua/fugit2/view/git_status.lua:1689: Command not found: Telescope
stack traceback:
	[C]: in function 'cmd'
	...are/nvim/lazy/fugit2.nvim/lua/fugit2/view/git_status.lua:1689: in function '_submit_fn'
	...im/lazy/fugit2.nvim/lua/fugit2/view/components/menus.lua:323: in function <...im/lazy/fugit2.nvim/lua/fugit2/view/components/menus.lua:320>
```

Additional context:
1. I'm using lazyvim and have telescope.nvim disabled (not uninstalled). Not sure if this is affecting things 🤔